docs/contributing.rst

```diff
@@ -5,12 +5,16 @@ All contributions to this repository are welcome via pull request on the `github
 
 The project follows the following guidelines:
 
-1. Use the base Python library unless completely necessary.
+1. Use the base Python library unless completely necessary. Numpy is ok.
```

Comment:
Is it worth wording this more along the lines of: please use what is currently in `requirements.txt`, if you need to use an external library perhaps discuss this with us? (Or maybe get rid of this completely?)